net/ip.go

```diff
@@ -208,6 +208,14 @@ func (n Network) Contains(nn NetworkNumber) bool {
 	return true
 }
 
+// Contains returns true if NetworkNumber is in range of Network, false
+// otherwise.
+func (n Network) Covers(o Network) bool {
```

Comment:
Add length check to make sure comparing IPv4 to IPv6 will return false, the reason being IPv4 and IPv6 example with same prefix could pass this test, but they don't cover each other
```
if len(n.Number) != len(n1.Number) {
	return false
}
```